puppetlabs/rototiller
# (qa-2542) Command `#add_switch`

commit eaa7107d2b1cfbc4ca24edf290e34dc68c0ec187
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Tue Oct 11 15:32:20 2016 -0700

```
(maint) set non-implemented acceptance tests to pending
```

commit 92ea240327dba68e2370be136a733cfc3d3c46ba
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Tue Oct 11 15:30:51 2016 -0700

```
(maint) fix env_var and command-env_var tests

* these tests snuck by with with false-positives
  * changes in `#execute_task_on` caught them
```

commit 40ab5543ed2b49898943ea9d8b0893a94da18569
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Tue Oct 11 15:03:09 2016 -0700

```
(QA-2542) add Command `#add_switch` markdown/human docs
```

commit 897e2cada83ea5b8339fecbf1eb7633313087017
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Mon Oct 10 17:07:25 2016 -0700

```
(QA-2542) acceptance: command add_switch

* add acceptance tests for command's `#add_switch`
* FIXME: changes to acceptance lib broke other tests
```

commit a2fa74feb6d98c5a5f7d8a2857900e0328e96f6f
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Mon Oct 10 16:57:20 2016 -0700

```
(maint) add docs:check task to view undocumented units
```

commit 7e2d8bcba436a6fc450d1c16175dba7b3c5b3f70
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Mon Oct 10 16:56:03 2016 -0700

```
(QA-2542) add yard docs for Command's `#add_switch`

* also add a bit more docs (`#param_collection`), to get our numbers up
```

commit 7ac0be4556cf113e0000293a84e0758c78766519
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Mon Oct 10 16:00:13 2016 -0700

```
(maint) fix documentation and Rake tasks

* add a link to CONTRIBUTING.md from README.md
* add architecture diagram, class_graph to CONTRIBUTING
* fix class_graph docs rake task
* remove class diagram from .gitignore
* use default yard dir, because: why not?
```

commit be52246675eee7c92d57e795c6a34b3faf347bff
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Mon Oct 10 15:59:26 2016 -0700

```
(maint) fix acceptance generate_host_config LAYOUT/TARGET
```

commit 77723acc444651b77d7a5bc526bef0164e3324b1
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Thu Oct 6 19:11:43 2016 -0700

```
(QA-2542) command add_switch
```

commit 2ed18309b4ca752d1657adcfbc028a529c0e4cce
Author: Eric Thompson eric.thompson@puppetlabs.com
Date:   Thu Oct 6 19:14:16 2016 -0700

```
(QA-2542) spec tests for command add_switch

* FIXME: this is almost exactly spec tests for add_env in some cases
  * we need to dry this thing out, but for now it's easy to follow
```
